docs/en_us/internal/testing.rst

```diff
@@ -552,19 +611,25 @@ To view test coverage:
 Python Code Style Quality
 ------------------
```

Comment:
sorry! when the HTML gets built, one of the requirements is that the length of the markup for a heading has to be at least as long as the heading string. So, the line of dashes has to extend at least as far as the y in Quality, or an error message will get thrown. It's not a big deal, it's one of those nit picky things. Thanks @vkaracic